src/util/http/MediaTypes.cpp

```diff
@@ -6,16 +6,21 @@
 
 #include <absl/strings/str_join.h>
 
-#include "../Exception.h"
-#include "../StringUtils.h"
-#include "../TypeTraits.h"
 #include "../antlr/ANTLRErrorHandling.h"
 #include "./HttpParser/AcceptHeaderQleverVisitor.h"
 #include "./HttpParser/generated/AcceptHeaderLexer.h"
 
 using std::string;
 namespace ad_utility {
 
+// The first media type in this list is the default, if no other type is
+// specified in the request. It's "application/sparql-results+json", as
+// required by the SPARQL standard.
+constexpr std::array SUPPORTED_MEDIA_TYPES{
+    MediaType::sparqlJson, MediaType::sparqlXml, MediaType::qleverJson,
+    MediaType::tsv,        MediaType::csv,       MediaType::turtle,
+    MediaType::octetStream};
```

Comment:
if you either first define these in a local `detail` namespace and then make it accesible, 
or (more modern) actually create them by an immediately invoked lambda, then you can benefit of
`using enum MediaType` to get  rid of the visual redundant overhead.